static/js/message_edit.js

```diff
@@ -509,6 +509,7 @@ function edit_message($row, raw_content) {
         case editability_types.NO:
             $message_edit_content.attr("readonly", "readonly");
             $message_edit_topic.attr("readonly", "readonly");
```

Comment:
This line is no longer needed now, so can simply be replaced by the new one

Reply:
Oops, sorry for missing on this, Removed. Thanks.